x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/filestructurefinder/FileStructureTestCase.java

```diff
@@ -1,89 +0,0 @@
-/*
```

Comment:
While this shows only a delete, it is effectively a file move (it is added later in the diff). The issue is, I think, that the formatting changes (enforced by our spotless settings), caused github to not recognize it as a simple file rename.